apps/packages/package_service.py

```diff
@@ -300,8 +301,10 @@ def replace_ref_in_package(self, package, old_ref_id, new_ref_id):
         non_root_groups = (group for group in package.get(GROUPS, []) if group.get(GROUP_ID) != ROOT_GROUP)
         for g in (ref for group in non_root_groups for ref in group.get(REFS, [])):
             if g.get(RESIDREF, '') == old_ref_id:
+                new_item = get_resource_service('archive').find_one(req=None, _id=new_ref_id)
                 g[RESIDREF] = new_ref_id
                 g['guid'] = new_ref_id
+                g[VERSION] = new_item[VERSION]
```

Comment:
Do we need to save the version of items in a unpublished package? Until it gets published a package always points to the last version of the referenced item.